bespin/collector.py

```diff
@@ -159,6 +159,7 @@ def convert_stack(path, val):
             env = configuration[["environments", environment]]
             if isinstance(env, six.string_types):
                 environment_as_dict = configuration[["environments", env]].as_dict()
+                env = configuration["environments", env]
```

Comment:
ah yeah, makes sense.

Though you need to pass a dictionary into the `__getitem__` syntax here.

    env = configuration[["environmets", env]]